core/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,103 @@
+#
```

Comment:
Here is the motive.

1. There is a log4j.proporties under conf/. But conf is not in default classpath when we do test in IDE. When you try to debug in IDE, you will find there is no log output.
2. In runtime, log4j will take the first loaded log4j.properties file. It wil still take conf/log4j.properties as higher priority. So there is no impact in production deployment.